packages/ui/src/components/va-counter/VaCounter.vue

```diff
@@ -396,9 +396,16 @@ export default defineComponent({
     }
   }
 
+
+
+
+
+
+
   .va-input-wrapper__field > *,
   .va-input-wrapper__container > * {
     margin-right: 0;
+    margin: 0.25rem;
```

Comment:
This also affects `.va-input-wrapper__field` (because it is a child of `.va-input-wrapper__container`) and adds additional margins to it. So we have to remove this. 
```suggestion
```

---
To solve this issue we need to add horizontal margins to elements with `va-counter__prepend-inner` and `va-counter__append-inner` classes. Their css rules are on lines 494...499. We can add their margins:
```css
  &__prepend-inner,
  &__append-inner {
    display: flex;
    align-items: stretch;
    height: 100%;
    margin: 0 0.5rem;
  }
  ```